lib/rust/parser/src/syntax/tree.rs

```diff
@@ -486,21 +496,35 @@ impl<'s> span::Builder<'s> for TextElement<'s> {
 
 // === Documentation ===
 
-/// A documentation comment.
+/// A documentation comment line.
+#[cfg_attr(feature = "debug", derive(Visitor))]
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Reflect, Deserialize)]
+pub struct DocLine<'s> {
+    /// The documentation.
+    pub docs:     DocComment<'s>,
+    /// Empty lines between the comment and the item.
+    pub newlines: Vec<token::Newline<'s>>,
+}
+
+impl<'s> span::Builder<'s> for DocLine<'s> {
+    fn add_to_span(&mut self, span: Span<'s>) -> Span<'s> {
+        span.add(&mut self.docs).add(&mut self.newlines)
+    }
+}
+
+/// A documentation comment line.
```

Comment:
We have two structures described as "Documentation comment line". How do they differ?